examples/python/hellostreamingworld/async_greeter_server.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python AsyncIO implementation of the GRPC hellostreamingworld.MultiGreeter server."""
+
+import asyncio
+import logging
+
+import grpc
+from hellostreamingworld_pb2_grpc import MultiGreeterServicer, add_MultiGreeterServicer_to_server
+from hellostreamingworld_pb2 import HelloRequest, HelloReply
+
+
+async def say_hello_generator(request):
+    for i in range(10):
+        yield HelloReply(message=f"Hello number {i}, {request.name}!")
+
+
+class Greeter(MultiGreeterServicer):
+
+    async def sayHello(
+            self, request: HelloRequest,
+            context: grpc.aio.ServicerContext) -> HelloReply:
+        logging.info(f"Serving sayHello request {request}")
+        async for msg in say_hello_generator(request):
```

Comment:
I would recommend to simplify port the message generation here.

Reply:
I've kept it that way to show how you can integrate with external generator (this was my first use case for this example)

Reply:
I see what you are proposing. This example file focuses on how to write a streaming gRPC server handler. Using an extra layer of generator doesn't seem to yield more helpful info for readers.